app/src/lib/databases/pull-request-database.ts

```diff
@@ -0,0 +1,22 @@
+import Dexie from 'dexie'
+
+export interface IPullRequest {
+  readonly id?: number
+  readonly repoId: number
+  readonly number: number
+  readonly title: string
+  readonly lastUpdate?: string
+}
+
+export class PullRequestDatabase extends Dexie {
+  public pullRequests: Dexie.Table<IPullRequest, number>
+
+  public constructor(name: string) {
+    super(name)
+
+    this.version(1).stores({
+      pullRequests:
+        'id++, repo_id, number, &[repo_id+number], [repo_id+updated_at]',
```

Comment:
I'm not sure we need all these indices? I imagine `repo_id` would be sufficient for now?

Reply:
I suppose this is left over from before I found out we were limited by the API.